From 9c8193c00cc6e14f342ab22471d3e1b996bd8a32 Mon Sep 17 00:00:00 2001
From: Alex Tu <alex.tu@canonical.com>
Date: Tue, 30 Jul 2019 14:30:48 +0800
Subject: [PATCH] fix: dkms script is missing function find_module(LP: #1838245
 )

1. correct the logic to compare with existed module. (https://github.com/dell/dkms/pull/90)
2. revert find_module which not be backported yet.
---
 dkms-2.3/dkms | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dkms-2.3/dkms b/dkms-2.3/dkms
index adce6f4..2293b49 100644
--- a/dkms-2.3/dkms
+++ b/dkms-2.3/dkms
@@ -781,8 +781,8 @@ check_version_sanity()
         fi
     fi
     set_module_suffix
-    read -a kernels_module < <(find_module "$lib_tree" "${4}")
-    [ -z $kernels_module ] || return 0
+    read -a kernels_module < <(find $lib_tree -name ${4}$module_suffix)
+    [ -z $kernels_module ] && return 0
 
     if [[ ${kernels_module[1]} ]]; then
         warn $"Warning! Cannot do version sanity checking because multiple ${4}$module_suffix" \
-- 
2.17.1

